Tests/SwiftParserTest/translated/DiagnoseInitializerAsTypedPatternTests.swift

```diff
@@ -84,7 +84,7 @@ final class DiagnoseInitializerAsTypedPatternTests: XCTestCase {
       diagnostics: [
         DiagnosticSpec(message: "unexpected initializer in pattern; did you mean to use '='?", fixIts: ["replace ':' with '='"])
       ],
-      fixedSource: "let f=/*comment*/[X]()"
+      fixedSource: "let f = /*comment*/[X]()"
```

Comment:
This is currently not working because we are overriding the trivia where I added the `isEmpty`. 

We can accept it or we can do something like "if presentNode.trailingTrivia end with space and the replacement trivia don't start with with a space, merge the trivia"

Reply:
The output I would expect for this is `let f =/*comment*/[X]()` (no space after `=`) because the comment already counts as trivia after the equal sign.